### DISCUSSION TITLE
Links in table cells

### DISCUSSION THREAD

**Discussion by vforget** [datapane/datapane#381] [Help]
### Summary

Is it possible to have the cell value in a table be a hyperlink?

The rough example below put the html verbatim into the cell.

### Additional information

```js
t['Link'] = "<a href='https://www.google.ca>google</a>"
dp.DataTable(t)
```

**Comment by mands** [comment]
Potentially yes, when using a Pandas dataframe that has been styled, see https://pandas.pydata.org/docs/user_guide/style.html for more guidance.

A styled `df` can then be passed to `dp.Table` and the styles, including links, will be applied.
